internal/pkg/aws/elbv2/elbv2.go

```diff
@@ -33,6 +35,44 @@ func New(sess *session.Session) *ELBV2 {
 	}
 }
 
+// ListenerRuleHostHeaders returns all the host headers for a listener rule.
+func (e *ELBV2) ListenerRuleHostHeaders(ruleARN string) ([]string, error) {
```

Comment:
Are you planning to have also a `path-pattern` method? so that we can concatenate the two and give an output to the clients?

Reply:
Hmm I was planning to continue using `RulePath` 🤔